test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -194,6 +259,19 @@ private void CompareWorkspaces(WorkspaceComparisonData a, WorkspaceComparisonDat
                 Assert.AreEqual(countA, countB, string.Format("One {0} node has {1} outports, while the other has {2}", a.NodeTypeMap[kvp.Key], countA, countB));
             }
 
+            foreach(var portkvp in a.PortDataMap)
```

Comment:
You added spaces after the "foreach" for all of the "foreach" statements below, space here?